packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/triggers/FirebaseNotificationTrigger.kt

```diff
@@ -10,14 +10,14 @@ import expo.modules.notifications.notifications.interfaces.NotificationTrigger
 /**
  * A trigger representing an incoming remote Firebase notification.
  */
-class FirebaseNotificationTrigger(private val remoteMessage: RemoteMessage) : NotificationTrigger {
+class FirebaseNotificationTrigger(val remoteMessage: RemoteMessage) : NotificationTrigger {
 
   private constructor(parcel: Parcel) : this(
     parcel.readParcelable(FirebaseNotificationTrigger::class.java.classLoader)
       ?: throw IllegalArgumentException("RemoteMessage from readParcelable must not be null")
   )
 
-  fun getRemoteMessage(): RemoteMessage = remoteMessage
+//  fun getRemoteMessage(): RemoteMessage = remoteMessage
```

Comment:
Do we need this function or we can just remove it?

Reply:
Temporary, I've fixed it in another PR